CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * **IMPROVEMENT** (by @lsaudon) Replace flutter_lints by very_good_analysis
 * **BREAKING** (by @lsaudon) Update dart sdk to 2.17.0 (flutter 3.0.0)
 * **BUGFIX** (by @imaNNeoFighT) Fix indicator out of range error in line chart, #1187
+* **FEATURE** (by @HTsuruo): Add `longPressDuration` optional property that allows to control the duration LongPress gesture occurs, #1128
```

Comment:
Here you need to mention those two issues in the description. You don't need to mention the current PR.

Reply:
Fixed it.
https://github.com/imaNNeoFighT/fl_chart/pull/1128/commits/e926194f408aa0c3426a11ead6792259fdd4649c